site2/docs/io-redis-sink.md

```diff
@@ -55,15 +55,81 @@ Before using the Redis sink connector, you need to create a configuration file t
 * YAML
 
     ```yaml
-    {
+    configs:
         redisHosts: "localhost:6379"
-        redisPassword: "fake@123"
-        redisDatabase: 1
+        redisPassword: "mypassword"
+        redisDatabase: 0
         clientMode: "Standalone"
         operationTimeout: 2000
-        batchSize: 100
+        batchSize: 1
         batchTimeMs: 1000
         connectTimeout: 3000
-    }
+    ```
+  
+### Usage
+
+This example shows how to write records to a Redis database using the Pulsar Redis connector.
+
+1. Start a Redis server.
+
+    ```bash
+    $ docker pull redis:5.0.5
+    $ docker run -d -p 6379:6379 --name my-redis redis:5.0.5 --requirepass "mypassword"
     ```
 
+2. Start a Pulsar service locally in standalone mode.
+
+    ```bash
+    $ bin/pulsar standalone
+    ```
+    Make sure the nar file is available at `connectors/pulsar-io-redis-{{pulsar:version}}.nar`.
```

Comment:
```suggestion
    Make sure the NAR file is available at `connectors/pulsar-io-redis-{{pulsar:version}}.nar`.
```

Reply:
Just copy this line from other sink examples and lowercase `nar` makes sence here.

Reply:
use correct captilization for proper nouns in tech writing
![image](https://user-images.githubusercontent.com/50226895/124531405-00a53680-de41-11eb-973b-4fcc1790070f.png)

Reply:
Okay, I mean other `md` examples use this style I just copy this line from there.

Reply:
In Pulsar docs, some contents are not in a correct writing style, we'd better avoid this.

Reply:
Done, I hava modified this.